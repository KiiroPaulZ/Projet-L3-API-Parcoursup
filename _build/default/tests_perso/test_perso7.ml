open Test_parcoursup_utils2

(* 
  Les candidat n'ont jamais réalisé de voeu et donc n'en reçoivent aucun.
*)
let session = nouvelle_session ()

let () =
  ajoute_candidat session ~nom_candidat:"Paul";
  ajoute_candidat session ~nom_candidat:"Margot";
  ajoute_formation session ~nom_formation:"Université Côte d'Azur" ~capacite:1;
  ajoute_formation session ~nom_formation:"Université de Toulon" ~capacite:1;
  ajoute_commission session 
    ~nom_formation:"Université Côte d'Azur" 
    ["Paul";"Margot"]; 
    ajoute_commission session 
    ~nom_formation:"Université de Toulon" 
    ["Paul";"Margot"]; 
  reunit_commissions session;
  nouveau_jour session;
  affiche_voeux_en_attente session "Paul";
  affiche_propositions_en_attente session "Paul";
  affiche_voeux_en_attente session "Margot";
  affiche_propositions_en_attente session "Margot";