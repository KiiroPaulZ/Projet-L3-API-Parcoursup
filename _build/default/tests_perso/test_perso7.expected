nouvelle session
ajoute candidat Paul
ajoute candidat Margot
ajoute formation Université Côte d'Azur (1 places)
ajoute formation Université de Toulon (1 places)
commission de classement de Université Côte d'Azur : [Paul; Margot]
commission de classement de Université de Toulon : [Paul; Margot]
réunit commissions
nouveau jour
Voeux en attente de Paul : {}
Propositions en attente de Paul : {}
Voeux en attente de Margot : {}
Propositions en attente de Margot : {}
